src/v2/view-builder/views/consent/EmailMagicLinkOTPTerminalView.js

```diff
@@ -36,7 +36,7 @@ const BaseEmailMagicLinkOTPTerminalView = View.extend({
 
 const OTPInformationTerminalView = BaseEmailMagicLinkOTPTerminalView.extend({
   template: hbs`
-  <h1 class='otp-value'>{{otp}}</h1>
+  <h1 class='otp-value no-translate'>{{otp}}</h1>
```

Comment:
These mocks were rendering a number on the UI that need not be translated.
  ```
  'terminal-return-otp-only-full-location.json',
  'terminal-return-otp-only-no-location.json',
  'terminal-return-otp-only-partial-location.json'
```

Reply:
![Uploading Screen Shot 2021-12-15 at 9.26.07 AM.png…]()

Reply:
![Screen Shot 2021-12-15 at 9 26 07 AM](https://user-images.githubusercontent.com/23267876/146236066-2be7c6e3-ae55-4f3a-8bb9-73cd47fd54a4.png)